app/pages/index.tsx

```diff
@@ -17,13 +19,23 @@ export default class Index extends Component<Props> {
         session {
           ...defaultLayout_session
         }
+        getReportingYear {
+          applicationOpenDate
+          applicationEndDate
+        }
       }
     }
   `;
 
   render() {
     const {query} = this.props;
     const {session} = query || {};
+    const date1: any = query.getReportingYear.applicationOpenDate;
+    const startDate: Date = new Date(Date.parse(date1));
+    const date2: any = query.getReportingYear.applicationEndDate;
+    const endDate: Date = new Date(Date.parse(date2));
+    const time1 = startDate.toDateString();
+    const time2 = endDate.toDateString();
     return (
```

Comment:
time1 & time2 are not very descriptive variable names. Something along the lines of parsedStartDate & parsedEndDate better describe the variables' purpose.